File: Folicon_Native/Classes/DArtTokenResponse.vb
Imports Newtonsoft.Json

Public Class DArtTokenResponse

        <JsonProperty("access_token")>
        Public Property AccessToken As String

        <JsonProperty("token_type")>
        Public Property TokenType As String

        <JsonProperty("expires_in")>
        Public Property ExpiresIn As Integer

        <JsonProperty("error")>
        Public Property ErrorType As String

        <JsonProperty("error_description")>
        Public Property ErrorDescription As String

        <JsonProperty("status")>
        Public Property Status As String
End Class

File: FoliCon/Models/Api/DArtTokenResponse.cs
namespace FoliCon.Models.Api;

public class DArtTokenResponse
{
    [JsonProperty("access_token")]
    public string AccessToken { get; set; }

    [JsonProperty("token_type")]
    public string TokenType { get; set; }

    [JsonProperty("expires_in")]
    public int ExpiresIn { get; set; }

    [JsonProperty("error")]
    public string ErrorType { get; set; }

    [JsonProperty("error_description")]
    public string ErrorDescription { get; set; }

    [JsonProperty("status")]
    public string Status { get; set; }
}